setup.go

```diff
@@ -201,17 +208,21 @@ func Setup(ctx context.Context, cfg Config, key crypto.PrivKey, dnsCache *cached
 	}
 
 	opts = append(opts, libp2p.Routing(func(h host.Host) (routing.PeerRouting, error) {
-		if cfg.RoutingV1 != "" {
-			routingClient, err := delegatedHTTPContentRouter(cfg.RoutingV1, routingv1client.WithStreamResultsRequired(), routingv1client.WithHTTPClient(httpClient))
+		var routingV1Routers []routing.Routing
+		for _, endpoint := range cfg.RoutingV1Endpoints {
+			rv1Opts := []routingv1client.Option{routingv1client.WithHTTPClient(httpClient)}
+			if endpoint != cidContactEndpoint {
+				rv1Opts = append(rv1Opts, routingv1client.WithStreamResultsRequired())
+			}
```

Comment:
:thought_balloon: Do we need this? I think boxo client implementation will work fine if `Accept: application/x-ndjson` request gets `application/json` response, and parse it as JSON and not NDJSON (thats what Kubo does, right?)

Reply:
Yes, I just kept it because it was what the code was doing before. What's cid.contact running? Does it support streaming?

Reply:
I'm merging this now, but we can clean this up in a quick PR. For now, we maintain the behaviour we had with cid.contact - if it turns out that cid.contact supports streaming, we should definitely update this.